### PR TITLE
[RFR] Allow live edition of dataset characteristics

### PR DESCRIPTION
**Pull request by djhi**
[Trello card](https://trello.com/c/0u5QmMM3)

### CODE REVIEW

File: src/api/models/publishedCharacteristic.js [hunk 1]
```diff
@@ -1,7 +1,18 @@
...
 
 export default (db) => {
     const collection = db.collection('publishedCharacteristic');
+
     collection.insertBatch = documents => chunk(documents, 100).map(data => collection.insertMany(data));
+
+    collection.updateValueById = (id, value) => collection.findOneAndUpdate({
+        _id: new ObjectID(id),
+    }, {
+        $set: { value },
```
Comment (ThieryMichel) [inline src/api/models/publishedCharacteristic.js:12]: where do you keep an array of versions?

**Review by ThieryMichel — CHANGES_REQUESTED** [review]

File: src/api/models/publishedCharacteristic.js (hunk 1 above)
Comment (djhi) [reply src/api/models/publishedCharacteristic.js:12]: not handled in this first pr

**Review by ThieryMichel — APPROVED** [review]

File: src/app/js/characteristic/sagas.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    const characteristics = response.map(r => r.value);
+    return yield put(updateCharacteristicsSuccess(characteristics));
+}
+
+export function* watchUpdateCharacteristics() {
+    yield takeLatest(UPDATE_CHARACTERISTICS, handleUpdateCharacteristics);
+}
+
+export default function* () {
+    yield fork(watchUpdateCharacteristics);
```
Comment (ThieryMichel) [inline src/app/js/characteristic/sagas.js]: useless fork use takeLatest directly

**Review by ThieryMichel — CHANGES_REQUESTED** [review]

File: src/app/js/characteristic/sagas.js (hunk 2 above)
Comment (djhi) [reply src/app/js/characteristic/sagas.js]: This is has been done like this almost everywhere.. I suggest we fix this in a dedicated pr then. I will fix this one though

**Review by ThieryMichel — APPROVED** [review]

**Pull request merged**